src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1604,78 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+lemma add_comm : I+J = J+I := by {rw [ideal.add_eq_sup, sup_comm, ← ideal.add_eq_sup]}
```

Comment:
Isn't this provable with `_root_.add_comm`?

Reply:
Good point, I've made the change.